README.md

```diff
@@ -49,33 +49,92 @@ Relinquish admin rights...
 
 ## Emergency
 
-# Deplyment and Bootstrapping Procedure
+# Deployment and Bootstrapping Procedure
 
 ...
-
-## Requirements:
-
+## Checklist:
```

Comment:
I don't think that the checklist is necessary here. What is not implemented should go into new Github issues.